### PR TITLE
(DOCSP-9751, DOCSP-10785, DOCSP-10786): Fixes

### PR DESCRIPTION
**Pull request by cbush**
## JIRA
https://jira.mongodb.org/browse/DOCSP-9751
https://jira.mongodb.org/browse/DOCSP-10785
https://jira.mongodb.org/browse/DOCSP-10786

## Staged
Realm app: https://docs-mongodbcom-staging.corp.mongodb.com/realm/bush/fixes/get-started/glossary.html

Create Realm app: https://docs-mongodbcom-staging.corp.mongodb.com/realm/bush/fixes/procedures/create-realm-app.html

Partitioning: https://docs-mongodbcom-staging.corp.mongodb.com/realm/bush/fixes/sync/partitioning.html

Backend: https://docs-mongodbcom-staging.corp.mongodb.com/realm/bush/fixes/tutorial/backend.html

Realm app: https://docs-mongodbcom-staging.corp.mongodb.com/realm/bush/fixes/tutorial/realm-app.html

### CODE REVIEW

File: source/tutorial/backend.txt [hunk 1]
```diff
@@ -29,18 +28,20 @@ This tutorial should take around 30 minutes.
...
-- You will learn how to execute serverless application logic with :doc:`MongoDB Realm Functions </functions>`.
-- You will learn how to execute logic in response to database and scheduled :doc:`triggers </triggers>`.
-- You will learn how to save and use static data, such as API keys, using :doc:`values and secrets</values-and-secrets>`.
-- You will learn how to use the GitHub API within your MongoDB Realm app.
-- You will learn how to view incoming requests and application events using :doc:`MongoDB Realm Application Logs</logs>`.
+You will learn how to:
+
+- Execute serverless application logic with :doc:`MongoDB Realm Functions </functions>`.
+- Respond to database and scheduled :doc:`triggers </triggers>`.
+- Save and use static data, such as API keys, using :doc:`values and secrets</values-and-secrets>`.
```
Comment (ccho-mongodb) [inline source/tutorial/backend.txt:35]: Nitpick: I think there was a discussion a month or two back about link-related directives that do not include a space between the link text and angled brackets not rendering correctly in nextgen, so this line and line 37 may cause issues when you migrate.

File: source/procedures/create-realm-app.txt [hunk 2]
```diff
@@ -14,7 +14,11 @@ Create a Realm App (Realm UI)
 
 Overview
 --------
-The following steps guide you through setting up a generic {+service+} application.
+
+The following steps guide you through setting up a generic {+service+}
```
Comment (ccho-mongodb) [inline source/procedures/create-realm-app.txt:18]: service => MongoDB Realm
app => Realm app
service app => MongoDB Realm Realm app

File: source/procedures/create-realm-app.txt [hunk 3]
```diff
@@ -14,7 +14,11 @@ Create a Realm App (Realm UI)
...
+{+app+}. The term :term:`"{+app+}" <{+app+}>` refers to the central,
+server-side configuration that functions as the backend for your mobile
```
Comment (ccho-mongodb) [inline source/procedures/create-realm-app.txt:20]: Nitpick: is it the configuration that functions as the backend or is it what it configures (MongoDB Realm)?

File: source/tutorial/backend.txt (hunk 1 above)
Comment (cbush) [reply source/tutorial/backend.txt:35]: @mongomoe Please take note.

**Pull request merged**